prelude/__internal__/List.mad

```diff
@@ -19,6 +19,14 @@ export last = (xs) => (#- xs.slice(-1)[0] -#)
 init :: List a -> List a
 export init = (xs) => (#- xs.slice(0, -1) -#)
 
+nth :: Number -> List a -> Maybe.Maybe a
+export nth = (i, xs) => (#- {
+  const x = xs[i];
+  return x === undefined
+    ? Maybe.Nothing()
+    : Maybe.Just(x);
+} -#)
```

Comment:
For this, it's fenced because it's multi-line? Or because it uses `undefined`? Or something else?

Reply:
Undefined mostly.